containers/opcua-device-gateway/entrypoint.sh

```diff
@@ -35,4 +35,4 @@ echo "  OPCUA_GATEWAY_IDENTIFIER: $OPCUA_GATEWAY_IDENTIFIER" >&2
 echo "  OPCUA_GATEWAY_NAME: $OPCUA_GATEWAY_NAME" >&2
 echo "Starting the opcua-device-gateway..."
 # shellcheck disable=SC2086
-exec /usr/bin/java $JAVA_OPTS -Dlogging.config=file:./logging.xml -Dspring.profiles.active=default,tenant -jar opcua-device-gateway.jar -Dspring.config.location=file:./application-tenant.yaml "$@"
+exec /usr/bin/java $JAVA_OPTS -Dlogging.config=file:./logging.xml -Dspring.profiles.active=default -jar opcua-device-gateway.jar  "$@"
```

Comment:
Nit. remove the double space.


```suggestion
exec /usr/bin/java $JAVA_OPTS -Dlogging.config=file:./logging.xml -Dspring.profiles.active=default -jar opcua-device-gateway.jar "$@"
```